### ISSUE TITLE
Suggestion - Add commands via discord

### ISSUE THREAD

**Issue by JasonInd** [QiCuiHub/discord-audio-pipe#26]
Was thinking it would be useful to add commands via discord to get the bot to work. This is because I can just leave on the python main file or executable to run on a server then control using discord commands.

So like !join (Voice channel name) to join a channel and !leave to leave.

**Comment by QiCuiHub** [comment]
yeah that's a good idea, I didn't think about the server use case

**Comment by Sandhu-Saabji** [comment]
Is there any update on this?
